config/puma.rb

```diff
@@ -1,7 +1,7 @@
 threads_count = ENV.fetch('MAX_THREADS') { 5 }.to_i
 threads threads_count, threads_count
 
-port        ENV.fetch('PORT') { 3000 }
+port        ENV.fetch('PORT') { 3000 }, ENV.fetch('HOST') { 'localhost' }
```

Comment:
I am not sure if we need an env variable for host.. I think command-line arguments will override these options anyway, so that could be used instead of HOST=0.0.0.0 or whatever.